docs/analytics_tools/storing_data.md

```diff
@@ -59,7 +59,7 @@ import geopandas as gpd
 import gcsfs
 import pandas as pd
 
-from calitp.storage import get_fs
+from calitp_data_infra.storage import get_fs
```

Comment:
everywhere that `storage.get_fs` is referenced I would import from `calitp-data` rather than `calitp-data-infra`; see https://github.com/cal-itp/calitp-py/issues/104

Reply:
Resolved in [e6f2773](https://github.com/cal-itp/data-infra/pull/2602/commits/e6f2773037116a38d5e1b6783fa7c63729318178)